pkg/platform/runtime/buildexpression/buildexpression.go

```diff
@@ -871,6 +871,25 @@ func (e *BuildExpression) removePlatform(platformID strfmt.UUID) error {
 	return nil
 }
 
+func (e *BuildExpression) SetDefaultTimestamp() error {
+	solveNode, err := e.getSolveNode()
+	if err != nil {
+		return errs.Wrap(err, "Could not get solve node")
+	}
+
+	for _, arg := range solveNode.Arguments {
+		if arg.Assignment == nil {
+			continue
+		}
+
+		if arg.Assignment.Name == "at_time" {
+			arg.Assignment.Value.Str = ptr.To("$at_time")
```

Comment:
I'm not sure if we should be using a constant here too, but it may be worth considering.